tests/snapshots.rs

```diff
@@ -54,21 +79,8 @@ struct Parameters {
     #[allow(dead_code)]
     #[serde(default)]
     image_bounds_check_policy: BoundsCheckPolicyArg,
-
-    #[cfg_attr(not(feature = "spv-out"), allow(dead_code))]
-    spv_version: (u8, u8),
-    #[cfg_attr(not(feature = "spv-out"), allow(dead_code))]
-    #[serde(default)]
-    spv_capabilities: naga::FastHashSet<spirv::Capability>,
-    #[cfg_attr(not(feature = "spv-out"), allow(dead_code))]
     #[serde(default)]
-    spv_debug: bool,
-    #[cfg_attr(not(feature = "spv-out"), allow(dead_code))]
-    #[serde(default)]
-    spv_adjust_coordinate_space: bool,
-    #[cfg_attr(not(feature = "spv-out"), allow(dead_code))]
-    #[serde(default)]
-    spv_separate_entry_points: bool,
+    spv: SpirvOutParameters,
```

Comment:
Should this still be `#[cfg_attr(not(feature = "spv-out"), allow(dead_code)]`?

Reply:
nicely spotted!